herddb-core/src/main/java/herddb/file/FileDataStorageManager.java

```diff
@@ -326,13 +332,15 @@ public void initTable(String tableSpace, String uuid) throws DataStorageManagerE
                 Bytes value = dataIn.readBytesNoCopy();
                 result.add(new Record(key, value));
             }
-            int pos = dataIn.getPosition();
```

Comment:
this getPosition must stay here, otherwise you are going to pass the wrong value to XXHash64Utils.hash at line 340